Formula/kryptor.rb

```diff
@@ -0,0 +1,24 @@
+class Kryptor < Formula
+  desc "Simple, modern, and secure file encryption and signing"
+  homepage "https://www.kryptor.co.uk/"
+  url "https://github.com/samuel-lucas6/Kryptor/archive/v3.1.0.tar.gz"
+  sha256 "e7372ac4ccc4676fa6079bcc7be257c3677003b8ec33df486a4d43ba8bd1f3af"
+  license "GPL-3.0-or-later"
+
+  depends_on "dotnet"
+
+  def install
+    os = OS.mac? ? "osx" : OS.kernel_name.downcase
+
+    system "dotnet", "publish", "src/KryptorCLI/KryptorCLI.csproj",
+           "-c", "Release",
+           "-r", "#{os}-x64",
```

Comment:
```suggestion
    os = OS.mac? ? "osx" : OS.kernel_name.downcase
    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s

    system "dotnet", "publish", "src/KryptorCLI/KryptorCLI.csproj",
           "-c", "Release",
           "-r", "#{os}-#{arch}",
```

Reply:
Actually, I've just realised this may cause a problem. There are libsodium binaries for Linux ARM/ARM64 but not macOS ARM64, which requires building as x64 but works fine on the M1 Macs.

Reply:
Can we use Homebrew `libsodium`? We have builds for macOS ARM64. Building as x64 doesn't work fine on M1 macOS, because we cannot assume that users have Rosetta installed.

Reply:
I don't know if that's possible. Where would the libsodium runtime be? It would need to be in the same folder as Kryptor. It's unfortunate that there isn't an ARM64 build in the NuGet package.

Reply:
It would be installed elsewhere, but we can symlink it to where kryptor expects it to be.

Reply:
Ok thought so. I guess that should work.

Reply:
I've added a 'depends on libsodium' line, but I have no idea how to do the symlink.